db-schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE "chat_message" (
+    "chat_id" uuid REFERENCES "item" ("id") ON DELETE CASCADE,          -- id of item to which chat is attached
+    "creator" uuid REFERENCES "member" ("id") ON DELETE SET NULL,       -- sender member
+    "created_at" timestamp NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc'),
+    "body" character varying(500)
```

Comment:
So this means the max chars is 500?

Reply:
Yes this is arbitrary for now, I've used `varchar` instead of `text` for performance reasons. I also thought this would avoid the possibility to spam the chat with very long messages. Would you prefer to use `text` instead?

Reply:
`varchar` is fine. Just thinking that then this number should be fixed and known in the back and front in order to show the user that they are running out of chars.